lib/compress/huf_compress.c

```diff
@@ -348,10 +353,11 @@ static void HUF_sort(nodeElt* huffNode, const unsigned* count, U32 maxSymbolValu
  *  `workSpace` must be aligned on 4-bytes boundaries, and be at least as large as a table of HUF_CTABLE_WORKSPACE_SIZE_U32 unsigned.
```

Comment:
Update this comment. The needed size of `workSpace` is now larger : `sizeof(HUF_buildCTable_wksp_tables)`.